src/components/Stages/index.tsx

```diff
@@ -138,7 +138,7 @@ function Stage() {
               <div className={style.stageData}>
                 <div className={style.stageTotalMoney}>
                   {stage?.stageContract.totalCommitted} ETH raised of{" "}
-                  {stage?.stage.stageGoal} ETH goal
+                  {stage?.stage?.stageGoal} ETH goal
```

Comment:
Why do we have stage within a stage? Also the naming stageGoal is redundant if we already know the context is a stage.

Reply:
Renamed it to stageData. The first stage is the name of the useState in the hook and the second stage is the name of the object inside of that useState hook. Changed stageGoal to goal